hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -55,63 +33,18 @@ public static void main(String[] args) {
    * 3) Start Name Node
    * 4) Start ZKFC Node
    */
-  @Override
-  public void launchTask(final ExecutorDriver driver, final TaskInfo taskInfo) {
-    executorInfo = taskInfo.getExecutor();
-    Task task = new Task(taskInfo);
-    log.info(String.format("Launching task, taskId=%s cmd='%s'", taskInfo.getTaskId().getValue(), task.getCmd()));
-    if (taskInfo.getTaskId().getValue().contains(HDFSConstants.NAME_NODE_TASKID)) {
-      nameNodeTask = task;
-      driver.sendStatusUpdate(TaskStatus.newBuilder()
-          .setTaskId(nameNodeTask.getTaskInfo().getTaskId())
-          .setState(TaskState.TASK_RUNNING)
-          .build());
-    } else if (taskInfo.getTaskId().getValue().contains(HDFSConstants.ZKFC_NODE_ID)) {
-      zkfcNodeTask = task;
-      driver.sendStatusUpdate(TaskStatus.newBuilder()
-          .setTaskId(zkfcNodeTask.getTaskInfo().getTaskId())
-          .setState(TaskState.TASK_RUNNING)
-          .build());
-    }
-  }
-
-  @Override
-  public void killTask(ExecutorDriver driver, TaskID taskId) {
-    log.info("Killing task : " + taskId.getValue());
-    Task task = null;
-    if (taskId.getValue().contains(HDFSConstants.NAME_NODE_TASKID)) {
-      task = nameNodeTask;
-    } else if (taskId.getValue().contains(HDFSConstants.ZKFC_NODE_ID)) {
-      task = zkfcNodeTask;
-    }
-
-    if (task != null && task.getProcess() != null) {
-      task.getProcess().destroy();
-      task.setProcess(null);
-    }
-    driver.sendStatusUpdate(TaskStatus.newBuilder()
-        .setTaskId(taskId)
-        .setState(TaskState.TASK_KILLED)
-        .build());
-  }
-
-  @Override
-  public void shutdown(ExecutorDriver d) {
-    // TODO(elingg) let's shut down the driver more gracefully
-    log.info("Executor asked to shutdown");
-    if (nameNodeTask != null) {
-      killTask(d, nameNodeTask.getTaskInfo().getTaskId());
-    }
-    if (zkfcNodeTask != null) {
-      killTask(d, zkfcNodeTask.getTaskInfo().getTaskId());
-    }
-  }
+  // TaskId will contain:
+  // NAME_NODE_TASKID
+  // ZKFC_NODE_ID
 
+  // TODO (llparse) this is a best-effort method, no guarantee of accurate transmission. rework without
+  // TODO (elingg) better handling in livestate and persistent state of zkfc task. Right now they are
+  // chained.
   @Override
   public void frameworkMessage(ExecutorDriver driver, byte[] msg) {
-    super.frameworkMessage(driver, msg);
+    /*super.frameworkMessage(driver, msg);
     String messageStr = new String(msg, Charset.defaultCharset());
-    File nameDir = new File(hdfsFrameworkConfig.getDataDir() + "/name");
+    File nameDir = new File(config.getDataDir() + "/name");
```

Comment:
Removing this logic will create problems with our failure scenarios. Need to make sure we keep this logic in place or update it and make sure it still works.